cmds/tui.c

```diff
@@ -554,6 +557,25 @@ static void copy_graph_node(struct uftrace_graph_node *dst,
 	}
 }
 
+static int tui_last_index(struct tui_window *win)
+{
+        int count = win->curr_index;
+        void *next, *prev = win->curr;
+
+        while(true)
+        {
+            next = win->ops->next(win, prev, false);
+            if (next == NULL)
+                return count;
+
+            count++;
+            if (win->ops->needs_blank(win, prev, next))
+                count++;
+
+            prev = next;
+        }
+}
```

Comment:
Looks like broken indentation (w/ expanded TAB)

Reply:
```bash
+static int tui_last_index(struct tui_window *win)$
+{$
+        int count = win->curr_index;$
+        void *next, *prev = win->curr;$
+$
+        while(true)$
+        {$
+            next = win->ops->next(win, prev, false);$
+            if (next == NULL)$
+                return count;$
+$
+            count++;$
+            if (win->ops->needs_blank(win, prev, next))$
+                count++;$
+$
+            prev = next;$
+        }$
+}$
+$
```
Apologize for m,y misunderstanding,
I'm not pretty sure which is the broken indentaion

(shown in `cat -A`)

Reply:
You have to use "TAB" instead of whitespaces. Please replace spaces to TABs.

Reply:
If your vim setting has `set expandtab` then remove the line in `.vimrc` or use `set noexpandtab`.